de-lib/src/main/java/org/iplantc/de/client/services/callbacks/NotificationCallbackWrapper.java

```diff
@@ -29,6 +32,10 @@ public void onSuccess(String result) {
         }
 
         setNotifications(notifications);
+        Splittable splitResult = StringQuoter.split(result);
+        if (splitResult.get("total") != null) {
+            total = Integer.parseInt(splitResult.get("total").asString());
```

Comment:
I think if `getTotal` was added to the `NotificationList` model, then the decoded `AutoBean` will have this value, and we won't need to manually parse it from the string `result`.

Reply:
```java
        NotificationList resultList = AutoBeanCodex.decode(notFactory, NotificationList.class, result).as();
        List<Notification> notifications = resultList.getNotifications();
        for (Notification n : notifications) {
            ... 
        }
        ... 
        total = resultList.getTotal();
```

Reply:
The total field needs to be a string in AutoBean otherwise, it doesn't get decoded. I think thats the reason it was parsed out using `Splittable`.

Reply:
😖 OK